jobs/config.json

```diff
@@ -11234,50 +11234,6 @@
       "sig-scalability"
     ]
   },
-  "ci-kubernetes-kubemark-5-gce-last-release": {
```

Comment:
Let's keep this for now and remove the other one.

Reply:
Done